dissect/target/plugin.py

```diff
@@ -528,14 +528,14 @@ def decorator(obj):
     return decorator
 
 
-def plugins(osfilter: Optional[type[OSPlugin]] = None) -> Iterator[PluginDescriptor]:
+def plugins(osfilter: Optional[OSPlugin] = None) -> Iterator[PluginDescriptor]:
```

Comment:
Nice, ran into this yesterday as well but didn't push the fix yet.

Isn't the type-hint still correct though?

Reply:
Yeah, that was a brain fart on my part. The check is wrong, but the typing is correct. I see a lot of testcases fail though, apparently I only tested the newly created test case :raised_eyebrow: